indy-didresolver/src/resolver.rs

```diff
@@ -17,6 +18,35 @@ use indy_vdr::pool::helpers::perform_ledger_request;
 use indy_vdr::pool::{Pool, PreparedRequest, RequestResult, TimingResult};
 use indy_vdr::utils::did::DidValue;
 
+#[derive(Serialize, Deserialize, PartialEq, Debug)]
+#[serde(rename_all = "camelCase")]
+pub enum Result {
+    DidDocument(DidDocument),
+    Content(Value),
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct ContentMetadata {
+    node_response: Value,
```

Comment:
Do we want to clarify what kind of transaction in indy we were checking here ? Imho it would make sense to add an additional field next to the nodeResponse like objectType?
Currently (for DID Documents), this would always be `NYM` but for DID URL referencing this could return other types like `CLAIM_DEF`

Reply:
Yes, this could be handy! I'll add that.